getting_started/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -21,12 +21,13 @@ styleguide.
 Code structure
 --------------
 
-Indentation
-~~~~~~~~~~~
-
-Indent type: Tabs *(editor default)*
+Formatting conventions
+~~~~~~~~~~~~~~~~~~~~~~
 
-Indent size: 4 *(editor default)*
+* Use line feed (**LF**) characters to break lines, not CRLF or CR. *(editor default)*
```

Comment:
Formatting really isn't something people should have to care about or do by hand. This is a reference for a code formatter, it's not a beginner tutorial. The editor already does that for you. The intro of the article already mentions that this is meant to help create a code formatter, among other things.

Reply:
Also, getting started =/= beginner. The step-by-step part of the guide is meant to be beginner friendly, but the scripting section is a reference for the programming languages Godot supports in the editor.

Reply:
Just because it's meant to work as a reference for a code formatter doesn't mean it can't also work well as an educational document for beginners.

Still, I think it's fine to mention these things since they are the editor default which means beginners can just ignore them. But also, if we want to provide some actionable advice to beginners, I could mention the command `git config --global core.autocrlf input` which fixes line endings automatically when committing using Git.

Reply:
Sure, the doc is still insightful. I was answering the part:

> be understood by someone who is writing his first computer program

File encoding and special characters are their own topics that take some time to learn and understand. To me, it's not the role of a style guide to be a tutorial for people writing their first computer program.